hbase/README.md

```diff
@@ -17,11 +17,13 @@ mvn clean package
 
 ### 3. Create a HBase table for testing
 
-```
-/HBASE-HOME-DIR/bin/hbase shell
+For best results, use the pre-splitting strategy recommended in [HBASE-4163](https://issues.apache.org/jira/browse/HBASE-4163):
 
-hbase(main):001:0> create 'usertable', 'family'
 ```
+hbase(main):001:0> create 'usertable', 'family', {SPLITS => (1..200).map {|i| "user#{1000+i*(9999-1000)/200}"}, MAX_FILESIZE => 4*1024**3}
+```
```

Comment:
The max filesize bit is questionable.

Should note that this makes 200 regions, and you should adjust dependg on cluster size. Should be a sizinh guide for regions per server in the hbase ref guide.